### PR TITLE
Additional image, music and video file formats for object preview

### PR DESCRIPTION
**Pull request by Alevsk**
Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts [hunk 1]
```diff
@@ -37,17 +37,54 @@ export const download = (
...
+  ];
   const textExtensions = ["pdf", "txt"];
-  const audioExtensions = ["wav", "mp3", "aac"];
-  const videoExtensions = ["mp4", "avi", "mpg", "webm"];
+  const audioExtensions = [
+    "wav",
+    "mp3",
+    "aac",
+    "m4a",
+    "flac",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts:61]: These three formats are not fully supported in all the browsers, that's why I removed them from original list

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts:61]: lossless xD

Comment (Alevsk) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts]: fixed

Comment (Alevsk) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/utils.ts]: i want to be able to listen to my music from console :p

**Review by bexsoft — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**